CHANGES.md

```diff
@@ -42,6 +42,10 @@ be compatible with this version,  specifically, those that ran with
  -  (Libplanet.Store) Added `FullNode.RemoveChild()` method.  [[#3576]]
  -  (Libplanet.Action) Added `IAccount.RemoveState()` interface method.
     [[#3577]]
+ -  (Libplanet.Explorer) Added `LegacyBencodexValueType` class that is
+    a copy of an old `BencodexValueType` with its name changed
+    for backwards compatibility.  Changed old `states` query
+    to use `LegacyBencodexValueType` instead of `BencodexValueType`.  [[#3579]]
```

Comment:
Should we add issue or PR numbers about previous feature

Reply:
How? I guess we can refer to #3560 maybe, but I don't think it'd help much. Also the whole point of this PR is to **not fix** the issue since we don't want to deal with the consequences of breaking the API. 😑